Game/Scripts/FinalBossScript.cpp

```diff
@@ -54,6 +54,10 @@ void FinalBossScript::Start()
 
 void FinalBossScript::Update(float deltaTime)
 {
+	if (isPaused)
+	{
+		return;
+	}
 	if (!target)
```

Comment:
```suggestion
	if (!target || isPaused)
```
change it only if you find it easier to understand